src/jdk.accessibility/windows/classes/com/sun/java/accessibility/internal/AccessBridge.java

```diff
@@ -3967,15 +3969,10 @@ private char getKeyChar(KeyStroke keyStroke) {
             debugString("[INFO]:   Shortcut is control character: " + Integer.toHexString(keyCode));
             return (char)keyCode;
         }
-        String keyText = KeyEvent.getKeyText(keyStroke.getKeyCode());
-        debugString("[INFO]:   Shortcut is: " + keyText);
-        if (keyText != null || keyText.length() > 0) {
-            CharSequence seq = keyText.subSequence(0, 1);
-            if (seq != null || seq.length() > 0) {
-                return seq.charAt(0);
-            }
-        }
-        return 0;
+
+        keyCode = keyStroke.getKeyCode();
+        debugString("[INFO]:   Shortcut is: " + Integer.toHexString(keyCode));
+        return (char) keyCode;
```

Comment:
```suggestion
        return (char)keyCode;
```
For consistency with the `return` statements above which have no space after the cast.